dataikuapi/dss/managedfolder.py

```diff
@@ -123,7 +123,8 @@ def upload_folder(self, path, folder):
             for file in files:
                 filename = os.path.join(root, file)
                 with open(filename, "rb") as f:
-                    self.put_file(os.path.join(path, os.path.relpath(filename, folder)), f)
+                    rel_posix_path = os.path.relpath(filename, folder).replace("\\", "/")
+                    self.put_file("{}/{}".format(path, rel_posix_path), f)
```

Comment:
This is not good on posix, as filenames _may_ contain those strange things that Humans who do not have the privilege to be geeks use, such as spaces and... \

You can do the following simple test:
- create  a `test_upload` dir containing a file named `a file_with\` (on GNU/Linux and OSX, you can do `touch a\ file_with\\` to create this one
- upload the dir using `upload_folder`
- see that the name of the uploaded file is `a file_with` :scream: 

Maybe we can perform the replacement iff `os.sep == '\'` ?

And, well, we should add a test for this case, in this PR (preferred) or in an additional test card, if there is not already one.

Reply:
You are right, thank you for pointing this !